default.nix

```diff
@@ -3,35 +3,41 @@
 , version ? "dev"
 }:
 
-pkgs.buildGoModule rec {
-  name = "morph-unstable-${version}";
-  inherit version;
+let
+  morph = pkgs.buildGoModule rec {
+    name = "morph-unstable-${version}";
+    inherit version;
 
-  nativeBuildInputs = with pkgs; [ go-bindata ];
+    nativeBuildInputs = with pkgs; [ go-bindata ];
 
-  src = pkgs.nix-gitignore.gitignoreSource [] ./.;
+    src = pkgs.nix-gitignore.gitignoreSource [] ./.;
 
-  buildFlagsArray = ''
-    -ldflags=
-    -X
-    main.version=${version}
-  '';
+    buildFlagsArray = ''
+      -ldflags=
+      -X
+      main.version=${version}
+    '';
 
-  vendorSha256 = "08zzp0h4c4i5hk4whz06a3da7qjms6lr36596vxz0d8q0n7rspr9";
+    vendorSha256 = "08zzp0h4c4i5hk4whz06a3da7qjms6lr36596vxz0d8q0n7rspr9";
 
-  postPatch = ''
-    go-bindata -pkg assets -o assets/assets.go data/
-  '';
+    postPatch = ''
+      go-bindata -pkg assets -o assets/assets.go data/
+    '';
 
-  postInstall = ''
-    mkdir -p $lib
-    cp -v ./data/*.nix $lib
-  '';
+    postInstall = ''
+      mkdir -p $lib
+      cp -v ./data/*.nix $lib
+    '';
 
-  outputs = [ "out" "lib" ];
+    outputs = [ "out" "lib" ];
 
-  meta = {
-    homepage = "https://github.com/DBCDK/morph";
-    description = "Morph is a NixOS host manager written in Golang.";
+    passthru = {
+      eval = args@{...}: (import (morph.lib + "/eval-machines.nix")) ({ inherit pkgs; } // args);
```

Comment:
I wonder if this should perhaps be `networkExpr: .... { inherit pkgs networkExpr; }` instead, so it would be used like `pkgs.morph.eval networkExpr` instead of `pkgs.morph.eval { inherit networkExpr; }`.